tyk-pro/scripts/bootstrap.sh

```diff
@@ -22,6 +22,31 @@ syntax_message(){
   exit 1
 }
 
+check_for_existing_organisations(){
+    EXISTING_ORGS=$(CURL_CA_BUNDLE=$CURL_CA_BUNDLE_TYK curl --silent --header "admin-auth: $TYK_ADMIN_SECRET" --header "Content-Type:application/json"  $DASHBOARD_HOSTNAME/admin/organisations 2>&1)
+    OWNER_NAMES=$(echo "$EXISTING_ORGS" | jq -r '.organisations | .[] | .owner_name | @base64')
+    if [[ -z "${OWNER_NAMES}" ]]
+    then
+      echo "No organisation present"
+    else
+      for row in $OWNER_NAMES; do
+        CURRENT_ORG_NAME=$(echo ${row} | base64 -d)
+        if [[ "$CURRENT_ORG_NAME" == "$TYK_ORG_NAME" ]]; then
+           echo "7"
+           return
+        fi
+      done
+      OWNERCNAMES=$(echo "$EXISTING_ORGS" | jq -r '.organisations | .[] | .cname | @base64')
+      for row in $OWNERCNAMES; do
+        CURRENT_ORG_CNAME=$(echo ${row} | base64 -d)
+        if [[ "$CURRENT_ORG_CNAME" == "$TYK_ORG_CNAME" ]]; then
+           echo "7"
```

Comment:
Ignore me if i am wrong but I believe this should be `echo "8"` based on the `echo "8"`'s error message.
```suggestion
           echo "8"
```

Reply:
Fixed in the next commit.

Reply:
Fixed.